src/org/javarosa/core/services/PropertyManager.java

```diff
@@ -104,7 +106,7 @@ public String getSingularProperty(String propertyName) {
         }
         if(retVal == null) {
             //#if debug.output==verbose
-            Std.out.println("Warning: Singular property request failed for property " + propertyName);
+            logger.warn("Singular property request failed for property {}", propertyName);
```

Comment:
Does this,
`//#if debug.output`
do anything? If not, let’s get rid of them as part of this change.

Reply:
I was really puzzled by these comments. It felt like c#'s regions...